modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,139 @@
+
+Mostly, these attributes indicate what other friendly units can actually pass through the portal -- that is, which kinds of unit that portal permits you to embark/disembark during the game.
+
+* `_(INF)_`: Permits Infantry units.
+* `_(4+)_`: Permits Vehicle units that have an Armour value of `4+` or lower.
+* `_({walker})_`: Permits Vehicle units that have the {walker} ability, regardless of their Armour value.
+
+We also indicate how the portal affects the arrival of detachments that you intend to use with it, through the `_(Roll)_` or `_(Auto)_` attributes.
+See <<Portal Reserves>> for more about this.
+
+TIP: We use `_(Roll)_` and `_(Auto)_` to represent different technologies for different factions.
+They are mutually exclusive, for both thematic and practical reasons -- so {portal} units in a given army list must all be of one of these types, never a mix of both.
+
+== Game effects of {portal}
+
+Where your army includes any units or objectives that have the {portal} ability:
+
+* Regardless of the scenario, you may keep any detachments that contain units that can use a portal out of the play area, as <<Portal Reserves>>.
+
+Where your unit or objective has the {portal} ability:
+
+* You can use this unit to move portal reserves detachments into play after they arrive.
+You can also move detachments that are in play between one portal and another.
+See <<portal-moves>>.
+
+[[portal-moves]]
+== Use a {portal} unit to move your forces
+
+You can use portals to move some of your forces around.
+Where you wish to move one of your detachments into or out of a portal, and it contains permitted units:
+
+* You may do this only in the Movement phase.
+// TODO: Is this stipulation necessary? On the one hand we're trying to apply standard transport rules for simplicity, but on the other hand this kind of stipulation sets up differences from those rules which complicates things again and creates logical gaps.
+* You may do this only if you give the detachment Advance or Assault orders.
+* Use the normal xref:main-rules:transportation.adoc[] rules to embark and disembark the detachment's units.
+footnote:[
+When you disembark units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
+In this case, just deduct 5 cm from the movement of each unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
+]
+// TODO: Do we need to mention something to the effect that portal reserves detachments either (a) already count as embarked so don't pay 5 cm penalty to embark, or (b) need to pay the penalty to embark, just as units already in play need to do? Also, given that for a detachment that is already in play you must embark and then immediately disembark from another portal, won't this get a bit complicated given that you must also pay a 5 cm penalty for *each* portal unit...? 
```

Comment:
For the first part, I think some nuance has been lost in the reorganisation. This section reads as if it only covers moving into one Portal and out of another, with no references to disembarking when arriving from Portal Reserves.

The Portal Reserves section only mentions disembarking when discussing the edge case where some of the units won't fit through a Portal.

For the second part, I think I'm missing something that makes it complicated. You're paying a 5 cm penalty for each Portal unit as you're embarking on one and disembarking from another, instead of paying 2 x 5 cm for embarking and disembarking from the same unit.

Reply:
> For the first part, I think some nuance has been lost in the reorganisation. This section reads as if it only covers moving into one Portal and out of another, with no references to disembarking when arriving from Portal Reserves.

Okay, I'll get my head back into it and look at that again soon. I think I see what you mean. I was trying to unpick some other issues, and this section is indeed phrased in terms of moving detachments that are already on the table, to clarify that side of things. But I can see that there's a bit missing on how to disembark from Portal Reserves.

> You're paying a 5 cm penalty for each Portal unit as you're embarking on one and disembarking from another, instead of paying 2 x 5 cm for embarking and disembarking from the same unit.

That's not what the Transportation rules state. They state that you pay 5cm to embark, and you pay 5cm to disembark — so if you do both then you pay 2x5cm=10cm. Again, is this "just like the standard Transportation rules", or not? 

Further, we have the particular case of arriving from Portal Reserves: If it should be exactly like standard Transportation rules then it still leaves room for debate on whether or not they count as 'already embarked'. I think our options are, in correlation with my TODO note here:

a. Make it clear that for units arriving from Portal Reserves the player must both embark and disembark and thus pay 2x5cm=10cm.
b. Make it clear that Portal Reserves count as already embarked and thus the player only has to disembark and thus pay 1x5cm=5cm.

I believe that (b) would be more consistent with how most people probably play when they deploy units that arrive in regular _Transport (X)_ units. But people may have different mental models for portals and Portal Reserves, and we should be take care to remove the room for debate.

Reply:
> That's not what the Transportation rules state. They state that you pay 5cm to embark, and you pay 5cm to disembark — so if you do both then you pay 2x5cm=10cm. Again, is this "just like the standard Transportation rules", or not?

I was replying to:

> won't this get a bit complicated given that you must also pay a 5 cm penalty for *each* portal unit...?

Which appeared to be talking about _the Portal units_ not the cargo units, so I explained the effect on the Portal units. 

Overall I think we're saying the same thing in different ways:

- Portal unit A pays 5 cm because the cargo units embark.
- Portal unit B pays 5 cm because the cargo units disembark.
- The cargo units pay 10 cm because they embark and disembark.


> Further, we have the particular case of arriving from Portal Reserves: If it should be exactly like standard Transportation rules then it still leaves room for debate on whether or not they count as 'already embarked'. I think our options are, in correlation with my TODO note here:
> 
> a. Make it clear that for units arriving from Portal Reserves the player must both embark and disembark and thus pay 2x5cm=10cm. b. Make it clear that Portal Reserves count as already embarked and thus the player only has to disembark and thus pay 1x5cm=5cm.
> 
> I believe that (b) would be more consistent with how most people probably play when they deploy units that arrive in regular _Transport (X)_ units. But people may have different mental models for portals and Portal Reserves, and we should be take care to remove the room for debate.

It's option (b). If you think a clarification is needed, please include one. I didn't specify in the original text because it was included in the way the Portal Reserves section was structured.

Reply:
> I was replying to:

>> [Also, given that for a detachment that is already in play you must embark and then immediately disembark from another portal,] won't this get a bit complicated given that you must also pay a 5 cm penalty for each portal unit...?

Yeah, I'd gotten slightly lost in the convolutions. A brief example to explain what I meant by 'complicated' in this context:

1. You move a detachment the contains {portal} unit A.
2. You move some other detachments.
3. You move a detachment that contains {portal} unit B.
4. You move detachment X with the intent to embark at B and disembark at A. However, did you think far enough ahead to leave both A and B with enough unused movement to 'pay' the 5 cm cost for each? Just how much did you move unit A five or ten minutes ago?

Assuming I have the correct interpretation, this seems relatively complex to me. No doubt some people's minds are untroubled by the amount of thinking ahead and memory that this involves, but that's not universally true — I'm pretty sure I'd find it troublesome. Hence I queried it.

If you disagree and don't think that there is cause for concern then we can go ahead with it and see what happens — the more so as this topic is 'experimental' anyway.

---

> It's option (b). [b. Make it clear that Portal Reserves count as already embarked and thus the player only has to disembark and thus pay 1x5cm=5cm.] If you think a clarification is needed, please include one. I didn't specify in the original text because it was included in the way the Portal Reserves section was structured.

I checked the draft as it was before I started editing, and I can't see how it was 'included'. [History — state of this file at commit 0b60f9d, just before I started editing](https://github.com/GameBrains/er-core/blob/0b60f9da7020d42d27de70b5518f1690670abad4/modules/experimental/pages/portal.adoc). You might argue that it's sort of implicit in the way that the text only mentioned disembarking, but that's a bit obscure in my opinion.

The thing that got me thinking in the first place boils down to this I think:

* For standard Reserves, I believe that we tend to picture the cargo units as assigned to a particular set of transports, and perhaps already in transit, and therefore already embarked by default.
* For Portal Reserves, we state that you can choose which portal to use to put a detachment into play. So, you might not picture the state of cargo units in quite the same way. You _might_ picture them mustering somewhere, near the 'entry' portal. When you decide which portal they will use as the 'exit' portal to disembark, that's when they actually embark at the entry portal — which could imply a cost to embark as part of the overall move. That's not the only way to think of it, but it's the image that popped into my mind as 'most obvious', and thus why I raised the question.

However, I may be overthinking this. I'll look again and maybe add something to make 'already embarked' explicit, or maybe skip it and return later if people actually raise questions about it.

Reply:
I'll go with this for now. Maybe we'll drop it later, or maybe future drafts will make it redundant anyway.

```suggestion
footnote:[
When you disembark units that you're bringing into play from Portal Reserves, treat them as already embarked -- so you only pay the 5 cm cost to disembark.
]
```